Example.iOS/Example.iOS.csproj

```diff
@@ -178,5 +173,33 @@
       <LastGenOutput>SecurityResources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Contents.json" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-20x20%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-20x20%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-20x20%403x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-29x29%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-29x29%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-29x29%403x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-40x40%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-40x40%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-40x40%403x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-57x57%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-57x57%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-60x60%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-60x60%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-60x60%403x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-72x72%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-72x72%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-76x76%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-76x76%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-76x76%403x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-83.5x83.5%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-Small-50x50%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-Small-50x50%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\ItunesArtwork%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\LaunchImages.launchimage\Contents.json" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcon.appiconset\Contents.json" />
```

Comment:
@wojta this line is the problem. We don't have this file in the repo

Reply:
Noo, it's there again :( There is no such thing as this file. But it keeps adding it.